### ISSUE TITLE
Entering the cost per KWh value higher than 100 (three digits) is not possible

### ISSUE THREAD

**Issue by olivercho** [teslamate-org/teslamate#1064]
**Describe the bug**
Hi, recently the Superchargers in Korea started to charge and I wanted to set the cost per kWh of my charge sessions as currently defined (313 KRW/kWh).
But as I tried to enter '313' into a charge session it didn't make any change and the value was not remembered.
After a few tries, I noticed that only less than 100 was allowed to enter and higher number than that was not available.
I guess the internal cost field in the database is set to use lower than 3 digits values, perhaps based on US prices.
Please change the setting to allow higher numbers, so that users can set the charge costs as same as defined in their currency values. (for me 313 KRW/kWh)  

**Expected behavior**

Allowed to enter the cost per kWh value as defined in KRW (three digits, 313 KRW/kWh)

**How to reproduce it (as minimally and precisely as possible)**:

1. Go to 'Charges'.
2. Click on one of session locations to create/edit geo-fence.
3. From the geo-fences page, choose the Cost type as "Per kWh".
4. Enter higher number than 100.
5. See error. It won't be saved.
6. Enter lower number than 100.
7. Now it will be saved properly.

**Relevant entries from the logs**

```
paste logs here
```

**Screenshots**

**Data**

**Environment**

- TeslaMate version: 1.20.1
- Type of installation: Docker
- OS TeslaMate is installed on: 
- User OS & Browser: MacOS High Sierra / Chrome
- Others:

**Comment by Beomi** [comment]
Same issue here,
Is it the issue due to the type of the DB column?

**Comment by Beomi** [comment]
But as I inspected this issue via Chrome, it even doesn't send any HTTP request which is over 100.

**Comment by olivercho** [comment]
@adriankumpf , I was able to adjust the DB column settings by following an article from a local Tesla community.
Below was the way I used which uses the docker-compose.yml.
Hopefully this is helpful to resolve this issue. 

`docker-compose exec database psql teslamate teslamate`
`ALTER TABLE charging_processes ALTER COLUMN cost TYPE numeric(8,2);`
`ALTER TABLE geofences ALTER COLUMN cost_per_unit TYPE numeric(7,4);`
`exit`
